lib/pure/collections/critbits.nim

```diff
@@ -333,10 +353,11 @@ iterator keys*[T](c: CritBitTree[T]): string =
 iterator values*[T](c: CritBitTree[T]): T =
   ## Yields all values of `c` in the lexicographical order of the
   ## corresponding keys.
+  ##
+  ## **See also:**
+  ## * `mvalues iterator <#mvalues.i,CritBitTree[T]>`_
   runnableExamples:
-    var c: CritBitTree[int]
-    c["key1"] = 1
-    c["key2"] = 2
+    let c = {"key1": 1, "key2": 2}.toCritBitTree
     var vals: seq[int]
```

Comment:
use sugar.collect to simplify test
(plus wherever relevant)

Reply:
For clarification, you mean for this?
```nim
var vals: seq[int]
for val in c.values:
  vals.add(val)
```
In that case, i think using `sequtils.toSeq` is even better.

Reply:
indeed, toSeq is good enough in this case; in other cases, if needed, use collect